omero/training-server/playbook.yml

```diff
@@ -305,6 +311,25 @@
         state: started
         restart_policy: always
 
+    - name: Run docker for omero-ms-zarr
+      become: yes
+      docker_container:
+        image: openmicroscopy/omero-ms-zarr:{{ omero_ms_zarr_release }}
+        name: omero_ms_zarr
+        env:
+          CONFIG_omero_db_host: localhost
+          CONFIG_omero_db_user: omero
+          CONFIG_omero_db_pass: omero
+          CONFIG_omero_db_name: omero
+          CONFIG_omero_data_dir: /OMERO
+        network_mode: host
+        exposed_ports: "8080"
+        published_ports: "8080:8080"
```

Comment:
```suggestion
```
[This doesn't do anything](https://docs.docker.com/network/host/) since you're running with `network_mode: host`